app-editors/vis/vis-0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="Vis aims to be a modern, legacy free, simple yet efficient vim-like editor."
```

Comment:
'Vis aims to be' is redundant (StupidDescription QA warning). Also remove the trailing dot since it is not supposed tobe a full sentence.